src/components/AppSidebar/AppSidebar.vue

```diff
@@ -387,20 +426,60 @@ export default {
 		},
 	},
 
+	updated() {
+		this.$nextTick(this.forceUpdateClamps)
+	},
+
 	methods: {
 		/**
 		 * Emit sidebar close event to parent component
 		 * @param {Event} e click event
 		 */
 		closeSidebar(e) {
+			/**
+			 * The sidebar is closing
+			 * @type {Event}
+			 */
 			this.$emit('close', e)
 		},
 
+		/**
+		 * Emit sidebar open event to parent component
+		 *
+		 * @param {HTMLElement} element the sidebar aside element
+		 * @param {Function} done the done appearing callback
+		 */
+		onAppear(element, done) {
+			/**
+			 * The sidebar is opening and the transition is in progress
+			 * @type {HTMLElement}
+			 */
+			this.$emit('open', element)
```

Comment:
ChristophWurst will love this, two proper separate events to manage before and after open :)